lib/kazoo/subscription.rb

```diff
@@ -0,0 +1,182 @@
+module Kazoo
+  class Subscription
+    attr_reader :timestamp, :version
+
+    def self.everything
+      create(/.*/)
+    end
+
+    def self.create(subscription, pattern: :white_list, timestamp: Time.now)
```

Comment:
Yeah, I don't really like the name, but this mirrors to field name in the JSON struct as it is stored in Zookeeper.